lib/js/atoms/Badge/Badge.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+	<div class="badge" :class="{ '-minor': color === BADGE_COLORS.MINOR }">
+		<div class="badge__icon">
+			<slot />
+		</div>
+		<div class="badge__label">{{ label }}</div>
+	</div>
+</template>
+
+<style lang="scss" scoped>
+@import '../../../styles/settings/colors';
+@import '../../../styles/settings/radiuses';
+@import '../../../styles/settings/typography';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/atoms/icons';
+
+.badge {
+	border-radius: $radius-s;
+	display: flex;
+	padding: $space-xxxxs $space-xxs;
+
+	&.-minor {
+		background-color: $color-minor-background;
+	}
+
+	&__icon {
+		display: flex;
+		align-items: center;
+		margin-right: $space-xxs;
+		height: $icon-xxs;
+		width: $icon-xxs;
+
+		& > img {
+			height: $icon-xxs;
+			width: $icon-xxs;
+		}
+	}
+
+	&__label {
+		@include textInfoS();
+		@include textBold();
+
+		color: $color-minor;
+		text-transform: uppercase;
+	}
+}
+</style>
+
+<script lang="ts">
+import { Value } from '../../utils/type.utils';
+
+import { BADGE_COLORS } from './Badge.consts';
+
+export default {
+	name: 'Badge',
+	props: {
+		label: {
+			type: String,
+			required: true,
+		},
+		color: {
```

Comment:
I'd name it `backgroundColor`, see https://bethinkteam.slack.com/archives/CFBLF5RGA/p1625137653015700?thread_ts=1625127129.002700&cid=CFBLF5RGA

Reply:
similar for the class name. It's very likely to clash with color modifiers in: https://github.com/bethinkpl/design-system/blob/eade93866eeddc4f3bd90bb08471dbfb8f52617e/lib/styles/modifiers/_colors.scss

Reply:
@rogatty I don't see any advantages of `backgroundColor` compare to `color`

Reply:
@kvas-damian I though we decided to keep BEM over css utils

Reply:
`backgroundColor` allows us to add new color features to this component in future. On the other hand many UI libraries use `color` as a prop name for similar components: https://vuetifyjs.com/en/components/chips/#usage

The problem with className clash I was refering to isn't negating current solution. I was thinking what will happen if someone introduces `-minor` in modifiers file. That's why I suggest to avoid this class name.

Reply:
> @rogatty I don't see any advantages of `backgroundColor` compare to `color`

Component API is more obvious with `backgroundColor`, I see this as an advantage. Someone in the future could assume that `color` also affects the slot media, which isn't true. It's just the background we're coloring.

Unless we plan to synchronize font color with background color at this component's level. In that case `color` would be better 🙂

Reply:
this minor requires 2 classes `&.-minor` so should have greater css score. But it will not protect for a `!important` syntax

Reply:
I can rename it into `badgeMinor` however it will against aBEM

Reply:
Second idea it to don't use `-` prefix in not aBEM classes

Reply:
IMO the exact classname is of a less importance. It's an internal detail which can be changed at any time.

The important decision is the prop name as it's a public API. IMO:
- If we're going to synchronize background and text color at this component's level then the prop should be `color`
- If we're not, then the prop should be `backgroundColor` which will later allow us to add `textColor` etc.

Reply:
I'll choose first option ;)

Reply:
Let's ask UX what they choose 🙂

Reply:
During the meeting we've agreed that the icon color would be computed in `BadgeWithIcon` and not here (documented in https://bethinkteam.slack.com/archives/CFBLF5RGA/p1625137653015700). Computing text color here doesn't seem to be consistent with it. That's why `backgroundColor` IMO is simpler and safer.

Reply:
UX has no time in this sprint https://bethinkteam.slack.com/archives/CFBLF5RGA/p1625144844018000?thread_ts=1625127129.002700&cid=CFBLF5RGA . Let's keep it like this and wait for designs with other colors